session_items.py

```diff
@@ -1,75 +1,34 @@
 from flask import session
-
-_DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
-]
-
+import requests
+from trello_config import trello_api_key, trello_token, trello_root_url, trello_board_id, trello_todo_list_id, trello_done_list_id
+from ToDoItem import ToDoItem
```

Comment:
`import ToDoItem` will suffice here

Reply:
I get a "TypeError: 'module' object is not callable" error if I do this. Doesn't it need to know what's being imported from the module?

Reply:
Yes that's a good point. This comment is obsoleted by the file name change anyway.